submissions/tests/test_models.py

```diff
@@ -164,3 +163,39 @@ def test_highest_score_hidden(self):
         highest = ScoreSummary.objects.get(student_item=item).highest
         self.assertEqual(highest.points_earned, 1)
         self.assertEqual(highest.points_possible, 2)
+
+    def test_override_highest_no_score(self):
+        item = StudentItem.objects.create(
+            student_id="score_test_student",
+            course_id="score_test_course",
+            item_id="i4x://mycourse/special_presentation"
+        )
+
+        # Override score with no score
+        Score.create_override_score(item, 10, 9)
+        highest = ScoreSummary.objects.get(student_item=item).highest
+        self.assertEqual(highest.points_earned, 9)
+        self.assertEqual(highest.points_possible, 10)
+
+    def test_override_highest_w_score(self):
+        item = StudentItem.objects.create(
+            student_id="score_test_student",
```

Comment:
Since we're using single quotes everywhere else, can we switch to them here for consistency?